Makefile

```
@@ -42,10 +42,11 @@ quality: requirements  # Run all quality checks
 	tox -e csslint,eslint,pycodestyle,pylint
 
 .PHONY: requirements
-requirements: requirements_js requirements_py  ## Install all required packages
+requirements: requirements_js requirements_py## Install all required packages
```

```suggestion
requirements: requirements_js requirements_py  ## Install all required packages
```

This should be reverted.